.circleci/config.yml

```diff
@@ -209,11 +246,50 @@ jobs:
           command: ghr -t $GITHUB_TOKEN -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME --replace $CIRCLE_TAG bin/
 
   publish-dev:
-    docker:
-      - image: cimg/go:1.14
+    executor: golang
     steps:
       - attach_to_workspace
       - setup_remote_docker
       - publish_docker_images:
           repo: opentelemetry-collector-contrib-dev
-          tag: ${CIRCLE_SHA1}
+          tag: ${CIRCLE_SHA1}-verified
```

Comment:
Why do we need this suffix? looks like we are not publishing images without it before running stability tests, right?

Reply:
We don't really need it but reasoning was as follows:

1. We've been pushing "unverified" builds historically so might be nice to differentiate between the two.
2. An alternate workflow was to publish images from main workflow but then add the verified tag on them from the stability tests workflow. 

We can remove the verified suffix. It's not really required at all if we only push verified images.

Reply:
I think it makes it cleaner if we push verified images only. It's good to be able always to refer to an image just by commit sha. 

The only concern is that if we want an image asap, we need to wait for stability test to finish. But in that case I'd rather add a suffix unverified images instead. What do you think? 

cc @tigrannajaryan

Reply:
Sounds good to me Dmitry

Reply:
Works either way for me (with slight preference for not adding any suffixes to image tags). We can revise if needed later.